plugin-server/src/worker/ingestion/persons/repositories/postgres-person-repository.ts

```diff
@@ -48,6 +53,162 @@ export class PostgresPersonRepository
         this.options = { ...DEFAULT_OPTIONS, ...options }
     }
 
+    private readonly protectedPropertyKeys: string[] = [
+        // Core person properties
+        'email',
+        'name',
+
+        // Properties automatically mapped from events to persons
+        '$app_build',
+        '$app_name',
+        '$app_namespace',
+        '$app_version',
+        '$browser',
+        '$browser_version',
+        '$device_type',
+        '$current_url',
+        '$pathname',
+        '$os',
+        '$os_version',
+        '$referring_domain',
+        '$referrer',
+        '$screen_height',
+        '$screen_width',
+        '$viewport_height',
+        '$viewport_width',
+        '$raw_user_agent',
+
+        // UTM and campaign tracking properties
+        'utm_source',
+        'utm_medium',
+        'utm_campaign',
+        'utm_content',
+        'utm_term',
+        'gclid',
+        'gad_source',
+        'gclsrc',
+        'dclid',
+        'gbraid',
+        'wbraid',
+        'fbclid',
+        'msclkid',
+        'twclid',
+        'li_fat_id',
+        'mc_cid',
+        'igshid',
+        'ttclid',
+        'rdt_cid',
+        'epik',
+        'qclid',
+        'sccid',
+        'irclid',
+        '_kx',
+
+        // Session and page tracking
+        '$session_id',
+        '$window_id',
+        '$pageview_id',
+        '$host',
+
+        // Identity and device tracking
+        '$user_id',
+        '$device_id',
+        '$anon_distinct_id',
+
+        // Initial/first-touch properties
+        '$initial_referrer',
+        '$initial_referring_domain',
+        '$initial_utm_source',
+        '$initial_utm_medium',
+        '$initial_utm_campaign',
+        '$initial_utm_content',
+        '$initial_utm_term',
+    ]
```

Comment:
suggestion: This list overlaps with the one from `plugin-server/src/utils/db/utils.ts`. Would it be possible to move both of them to a file in the persons module and instead of exporting the lists, export a couple of functions, like `canTrimProperty(propertyName)` or something like this?